runtime/dart_init.cc

```diff
@@ -188,11 +193,13 @@ Dart_Isolate ServiceIsolateCreateCallback(const char* script_uri,
   // No VM-service in release mode.
   return nullptr;
 #else   // FLUTTER_RUNTIME_MODE
-  tonic::DartState* dart_state = new tonic::DartState();
-  Dart_Isolate isolate = Dart_CreateIsolate(
-      script_uri, "main", g_default_isolate_snapshot_data,
-      g_default_isolate_snapshot_instructions, nullptr, dart_state, error);
+  blink::UIDartState* dart_state = new blink::UIDartState(nullptr, nullptr);
```

Comment:
Do you need "blink" namespace qualifier here? I think above you referenced UIDartState without namespace.

Reply:
Apparently I do not.